bot/resources/evergreen/py_topics.yaml

```
@@ -1,25 +1,29 @@
-# Conversation starters for python-related channels.
+# Conversation starters for Python-related channels.
 
 # python-general
-267624335836053506:
+704362672644423840:
     - What's your favorite PEP?
     - What's your current text editor/IDE, and what functionality do you like about it the most when programming in Python?
     - What functionality is your text editor/IDE missing for programming Python?
     - What parts of your life has Python automated, if any?
-    - Which python project are you the most proud of making?
+    - Which Python project are you the most proud of making?
     - What made you want to learn Python?
     - When did you start learning Python?
     - What reasons are you learning Python for?
     - Where's the strangest place you've seen Python?
     - How has learning Python changed your life?
+    - Is there a package you wish existed but doesn't? What is it?
+    - What feature do you think should be added to Python?
+    - Has Python helped you in school? If so, how?
+    - What was the first thing you created with Python?
 
 # async
 630504881542791169:
     - Are there any frameworks you wish were async?
     - How have coroutines changed the way you write Python?
 
 # computer-science
-650401909852864553:
+704406535509639209:
```

This isn't the correct ID for `#computer-science`. I'm guessing you committed the test server channel IDs?
```suggestion
650401909852864553:
```